x-pack/plugin/inference/build.gradle

```diff
@@ -4,10 +4,6 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-
-import org.apache.tools.ant.taskdefs.condition.Os
```

Comment:
My ide keeps removing these claiming they're not used. If the build fails I'll add them back.